src/js/pages/ArbitrationLicense/components/Balance.jsx

```diff
@@ -0,0 +1,31 @@
+import React, {Component, Fragment} from 'react';
+import PropTypes from 'prop-types';
+import {withNamespaces} from 'react-i18next';
+import classnames from 'classnames';
+
+import SNTIcon from '../../../../images/tokens/SNT.png';
+
+class YourSNTBalance extends Component {
```

Comment:
Don't we already have a Balance component for the License? We can maybe re-use it?